tests/std/tests/P0088R3_variant/env.lst

```diff
@@ -32,7 +32,7 @@ PM_CL="/BE /c /EHsc /MD /std:c++latest /permissive-"
 PM_CL="/BE /c /EHsc /MDd /std:c++17 /permissive-"
 PM_CL="/BE /c /EHsc /MT /std:c++20 /permissive-"
 PM_CL="/BE /c /EHsc /MTd /std:c++latest /permissive-"
-PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing /EHsc /MD /std:c++20 /permissive-"
+PM_COMPILER="clang-cl" PM_CL="-fno-ms-compatibility -fno-delayed-template-parsing /EHsc /MD /std:c++latest /permissive-"

```

Comment:
AFAIK `/permissive-` is meaningless for clang and `cl /BE`. I'm only touching this file to sync it back up with `usual_17_matrix` like the comment says, I don't intend to make any other changes.

That said, we need to make a pass over the env lists one of these days to remove `/permissive` and `/permissive-` where they are the defaults and/or meaningless.

Reply:
clang understands `/permissive` and `/permissive-`  since clang 13: https://github.com/llvm/llvm-project/commit/c70b0e808da8e1650f3ee426698a8b87c94b8910

Reply:
I'm not sure what if anything `/Zc:twoPhase-` means to clang-cl, since it doesn't accept my test case:
```c++
namespace N {
    struct S {};
}

template <class T>
constexpr bool f(T) {
    return true;
}

template <class T>
constexpr bool g(T x) {
    return f(x);
}

constexpr bool f(N::S) {
    return false;
}

static_assert(!g(N::S{}), "");
```
I'm unable to determine if `/std:c++20` and `/std:c++latest` imply `/permissive-`.